repos/c2sm/packages/py-gt4py/package.py

```diff
@@ -49,6 +50,7 @@ class PyGt4py(PythonPackage):
     depends_on('py-numpy@1.24.2: ~blas ~lapack', type=('build', 'run'))
     depends_on('py-packaging@20.0:', type=('build', 'run'))
     depends_on('py-pybind11@2.5:', type=('build', 'run'))
+    depends_on('py-nanobind@1.4.1:', when="@1.1.3:", type=('build', 'run'))
```

Comment:
version `main` needs to depend on py-nanobind11 too.
Something like `when=@1.1.3:,main` should do the trick

Reply:
I think this is not required. Per spack rules main is greater than any numbered version. https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison